schema/360-giving-schema.json

```diff
@@ -735,7 +735,7 @@
             "number",
             "null"
           ],
-          "description": "The latitude of a point location",
+          "description": "The latitude of a point location. Using [WGS 84](https://en.wikipedia.org/wiki/World_Geodetic_System) standard, for example 51.501502",
```

Comment:
If one was being very pedantic, one might argue that this is a backwards-incompatible change because previously-allowed values are no longer allowed.

However, I find it incredibly unlikely that anyone is using anything other than WGS 84, so I doubt this is a real issue. I like the provision of an example.

Reply:
thanks @robredpath I raised exactly this question in particular in a planio ticket - [30810](https://opendataservices.plan.io/issues/30810) - as I believe this isn't a backwards incompatible change to the schema, but a clarification of guidance. Let me know what you think.

Reply:
@KDuerden I agree - for all practical purposes, it's backward-compatible and so it's fine. I will buy, and eat, a hat if this change actually breaks anyone's use case. 

The WGS 84 vs EPSG 4326 question is an even smaller nit to pick, but we might as well get it right. @Lathrisk will be able to tell us what to do.

Reply:
Are there any comments about EPSG that have an impact on this PR or are they issues for separate consideration?